main.rb

```diff
@@ -0,0 +1,20 @@
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
+end
+def foo
+  eval(foo)
+end
+
+class AAA
+        def initialize
+          
+        end
+end
+
```

Comment:
__[development][rubocop]__
* Line is too long. [120/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->